src/main/java/bral/projectile/SimpleDocumentListener.java

```diff
@@ -0,0 +1,21 @@
+package bral.projectile;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+
+@FunctionalInterface
+public interface SimpleDocumentListener extends DocumentListener {
+    void update(DocumentEvent e);
```

Comment:
Where did you see this?

Reply:
I did this from the Stackoverflow that was emailed to us as instructions.